packages/forms-web-app/src/dynamic-forms/has-questionnaire/questions.js

```diff
@@ -134,11 +134,12 @@ exports.questions = {
 			}
 		]
 	}),
-	// siteNoticeUpload: new MultiFileUploadQuestion({
-	// 	title: 'Site notice',
-	// 	question: 'Upload site notice',
-	// 	fieldName: 'site-notice-upload'
-	// }),
+	uploadSiteNotice: new MultiFileUploadQuestion({
+		title: 'Upload the site notice',
+		question: 'Upload the site notice',
+		fieldName: 'upload-site-notice',
+		validators: [new RequiredValidator('You must add your documents')]
```

Comment:
I'll put this on this PR as well in case you see this one first:

for multifile validation, this line should now read:
`validators: [new RequiredFileUploadValidator(), new MultifileUploadValidator()]`